Cargo.toml

```diff
@@ -24,6 +24,9 @@ clipboard_macos = { version = "0.1", path = "./macos" }
 clipboard_x11 = { version = "0.3", path = "./x11" }
 clipboard_wayland = { version = "0.2", path = "./wayland" }
 
+[target.'cfg(not(any(all(unix, not(any(target_os = "macos", target_os = "ios", target_os = "android", target_os = "emscripten"))), target_os = "windows", target_os = "macos", target_os = "ios", target_os = "android")))'.dependencies]
+clipboard_dummy = { version = "0.3", path = "./dummy" }
```

Comment:
I don't think we need a new `dummy` crate, since we aren't really using it anywhere!

Reply:
I was using iced_native for wasm. But iced_native code is really coupled with this crate, I can't compile iced to wasm. So I just patch this crate by adding a dummy platform.

I think it's a good thing to just add a dummy platform on this crate for an unsupported platform rather than modifying the GUI library that depends on this crate.

Reply:
I meant that the `clipboard_dummy` crate seems unnecessary.

The `dummy` platform in the root crate is fine!